lib/winapi/winbase.h

```diff
@@ -2,6 +2,7 @@
 #define __WINBASE_H__
 
 #include <windef.h>
+#include <winnt.h>
```

Comment:
I'm not sure if this should be included from here, or from windef or elsewhere.

I don't remember why I put this here.